Sources/MessagingInApp/MessagingInApp.swift

```diff
@@ -21,8 +21,8 @@ public class MessagingInApp: ModuleTopLevelObject<MessagingInAppInstance>, Messa
     // constructor that is called by test classes
     // This function's job is to populate the `shared` property with
     // overrides such as DI graph.
-    override init(implementation: MessagingInAppInstance?, sdkInitializedUtil: SdkInitializedUtil) {
-        super.init(implementation: implementation, sdkInitializedUtil: sdkInitializedUtil)
+    override init(implementation: MessagingInAppInstance?) {
```

Comment:
If I remember correctly, `ModuleTopLevelObject` was created to initialize modules in a way that worked with SDK initialization. By making push and in-app no longer tied to SDK initialization, I think that we can make `MessagingPush` and `MessagingInApp` no longer extend `ModuleTopLevelObject` and possibly even remove `ModuleTopLevelObject` entirely?

Reply:
Yes, we might be able to do that. But I'm a bit hesitant to remove this entirely at this point. I want to hold it till the feature modules are actually fully independent, we can then reevaluate this and remove if no longer needed. I have added a TODO so we don't forget this.